devices.js

```diff
@@ -6877,18 +6877,19 @@ const devices = [
         },
     },
 
-    // Gira
+    // Insta/Gira/Jung
     {
         zigbeeModel: [' Remote'],
         model: '2430-100',
-        vendor: 'Gira',
+        vendor: 'Insta/Gira/Jung',
```

Comment:
Are the Gira and Jung white-labelled devices? if so we can mark them as such
- example: https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices.js#L1044
- See https://www.zigbee2mqtt.io/information/supported_devices.html#zemismart ZM-CSW002-D how it shows up in the docs.

Reply:
Yes, they are both white-labelled devices originally manufactured by [Insta](https://www.insta.de/instastorefront/en/). If you're ok with https://github.com/Koenkk/zigbee-herdsman-converters/pull/1082/commits/b74264fd32316b713cc7274dfb1ebe8278a526c3, then I'll file another PR to change the device model in https://github.com/Koenkk/zigbee2mqtt/blob/master/lib/extension/homeassistant.js#L944 also.